timemachine/md/exchange/exchange_mover.py

```diff
@@ -355,8 +355,29 @@ def swap_vi_into_vj(
         log_weights_after_full = np.array(log_weights_after_full)
         log_weights_after = log_weights_after_full[vj_plus_one_idxs]
 
+        # directionality is going from region i into region j
+        # take care of asymmetric boundary conditions
+        if len(vi_mols) == self.num_waters:
+            # region i has every water, so fwd probability is 100%, rev probability is 50%
+            g_fwd = 1.0
+            g_rev = 0.5
+        elif len(vi_mols) == 1:
+            # region i has only one water, so fwd probability is 50%, rev probability is 100%
+            g_fwd = 0.5
+            g_rev = 1.0
```

Comment:
Is there any nuance of this 100%? For example lets say you have a buckyball as the inner volume that has a methane (no waters) inside and there is no space for the water, in that case the probability wouldn't be 100% to accept that move. Or if the radius of the inner sphere contained the carbons you could have a clash with the new water.

Reply:
The probabilities above indicate the *proposal* probability, not the *acceptance* probability. The clashiness is taken care of by pi(x), i.e. the probability assigned to the stationary distribution.